sonobuoy/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# This script is meant to be run from inside the Sonobuoy pod
+bin/run-all
+
+cp -r ./reports /tmp/results
```

Comment:
Consider adding the `reports` folder to `.gitignore`, so that git won't track that folder during multiple test runs.